repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,104 @@
 
 include::../_includes/in-progress.adoc[]
 
-<<add-header.adoc#,Add Header Filter>>::
-    Enables users to add new headers and header values to a request and/or response.
+== <<add-header.adoc#,Add Header Filter>>
```

Comment:
I agree with not letting the tool dictate what we can do, but I also hate warnings where avoidable. So I'm conflicted. 😉 

These actually started as a [labeled list](http://asciidoctor.org/docs/user-manual/#labeled-list) but we changed them to sections to get a TOC. I believe the primary reason for doing so was so that we had a TOC consistently on every page. I could be mistaken, though. I would note that a TOC also buys us quick links where less scrolling is required (since description text is not present). That may be useful for users that already know what filter they are looking for.